lib/datasource/sbt-plugin/util.ts

```diff
@@ -1,12 +1,25 @@
+import url from 'url';
```

Comment:
Nope, use our url util or global URL object.

Reply:
The import is done in `lib/datasource/maven/util.ts` like this, but no problem, I'll also change it there.

Reply:
https://github.com/renovatebot/renovate/pull/9109 😉

Reply:
Done :)

Reply:
https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#resolve-review-comments-instead-of-commenting 😉